src/bp-templates/bp-nouveau/buddypress/common/search/profile-search.php

```diff
@@ -222,7 +222,7 @@
 					$km          = __( 'km', 'buddyboss' );
 					$miles       = __( 'miles', 'buddyboss' );
 					$placeholder = __( 'Start typing, then select a location', 'buddyboss' );
-					$icon_url    = plugins_url( 'bp-profile-search/templates/members/locator.png' );
+					$icon_url    = BP_PLATFORM_API . 'bp-core/profile-search/templates/members/locator.png';
```

Comment:
@shanebp  same here it should be a like this `buddypress()->plugin_url` instead of `BP_PLATFORM_API`

```suggestion
					$icon_url    = buddypress()->plugin_url  . 'bp-core/profile-search/templates/members/locator.png';
```

Reply:
I have made the changes you suggest. 
It is okay now?